### ISSUE TITLE
CPU Load naming in servinfo ReST call

### ISSUE THREAD

**Issue by indigo423** [nextcloud/serverinfo#36]
It would be nice to know which value is 1 minute, 5 minute and 15 minute load. 

```
<cpuload>
<element>2.76806640625</element>
<element>1.99267578125</element>
<element>1.67626953125</element>
</cpuload>
```

should be similar to active users

```
<activeUsers>
<last5minutes>1</last5minutes>
<last1hour>1</last1hour>
<last24hours>1</last24hours>
</activeUsers>
```

**Comment by indigo423** [comment]
As I see it is a PHP internal call http://php.net/manual/de/function.sys-getloadavg.php

**Comment by indigo423** [comment]
Just as a side node, I didn't had any problems in OpenNMS, cause we use XPath[1] and this is not complicated. The change would make it for other monitoring tools a little bit easier where people have to parse the XML file manually.

[1] https://github.com/opennms-config-modules/nextcloud/blob/master/xml-datacollection/nextcloud-serverinfo.xml#L6

**Comment by nickvergessen** [comment]
Yeah, problem is that we can not rename the elements easily, since it will break existing endpoint consumers.
But we could of course duplicate the elements for the time-being

**Comment by nickvergessen** [comment]
Feel free to make a pull request to do this.
